### DISCUSSION TITLE
option to check for standard tasmota bin

### DISCUSSION THREAD

**Discussion by Shiroe93** [benzino77/tasmocompiler#372] [Q&A]
hello  two quick questions:
1 which option i have to check to obtain the standard tasmota.bin (the base version)? i ask because from there i only need to add one sensor with custom parameters 
2) what does the home assistant flag add specifically?
thank you for this repo it's very useful

**Comment by benzino77** [comment]
Hi there!
1. Hard to say. Default settings are "bare bones" of Tasmota. It includes almost nothing. Then you customize your binary by checking features and adding "custom parameters"
2. Have a look here: https://github.com/benzino77/tasmocompiler/blob/master/FEATURES_DESC.md

**Reply by Shiroe93** [reply]
I've already seen that page you linked but unless I'm missing something the "home assistant" flag is not described

**Reply by benzino77** [reply]
You are right. Have a look here: https://github.com/benzino77/tasmocompiler/blob/26ebd63917a73baaafb3cdb93bd9e290a1d8eab4/src/components/AppStepper/FeaturesStep/AvailableFeatures.js#L209

Checking this feature just adds:
```c
#define USE_HOME_ASSISTANT
```

**Reply by Shiroe93** [reply]
![Screenshot_20230817_125548_GitHub.png](https://github.com/benzino77/tasmocompiler/assets/56954468/2ead2a56-e9c6-4007-8970-c8ac85ccdf2d)

This is what I see from your link but I don't understand what it entails
What changes? Wich feature are added that they wouldn't be there if I don't activate the flag?

**Reply by benzino77** [reply]
If you activate the flag "Home assistant", the 
```c
#define USE_HOME_ASSISTANT
```
is added to your `user_config_override.h` file


To find out which options are turned on by default you should check resulting `user_config_override.h` file (you can download it at the end of the compilation) or check the file I provided and see which features has `value: true` (which means those are turned on by default).

**Reply by benzino77** [reply]
TasmoCompiler tweaks Tasmota by adding things to `platformio_override.ini` and  `user_config_override.h` files only - NOTHING else. Both files can be downloaded at the end of the compilation, so you have an ability to see what is inside of them and even use them to compile your binary from command line.
